pages/profile/[vin].js

```diff
@@ -1,20 +1,57 @@
+import EditCarForm from "@/components/EditCarForm";
 import Image from "next/image";
 import { useRouter } from "next/router";
-import Cars from "../../db/db.json";
+import { useState } from "react";
+import initialCars from "../../db/db.json";
+
 export default function CarDetails() {
+  const [isEditing, setIsEditing] = useState(false);
   const router = useRouter();
   const { vin } = router.query;
-  const activeCar = Cars.find((car) => {
-    return car.VIN === vin;
+
+  const [cars, setCars] = useState(initialCars);
+
+  const activeCar = cars.find((car) => {
+    return { ...(car.VIN === vin), milage: 0, plate: "" };
   });
```

Comment:
Your find function does not work properly work, since it will always return true and will always return the first car.

Reply:
Done!